machine_learning_hep/analysis/analyzer_jet.py

```diff
@@ -1978,51 +1961,51 @@ def jetsystematics(self):
             for ibinshape in range(self.p_nbinshape_gen):
                 sys_up_z=[]
                 sys_down_z=[]
-                error_full_up=0
-                error_full_down=0
+                error_full_up = 0
+                error_full_down = 0
                 for sys_cat in range(len(self.systematic_categories)):
-                    error_var_up=0
-                    error_var_down=0
-                    count_sys_up=0
-                    count_sys_down=0
+                    error_var_up = 0
+                    error_var_down = 0
+                    count_sys_up = 0
+                    count_sys_down = 0
                     for sys_var in range(self.systematic_variations[sys_cat]):
                         error = input_histograms_sys[ibin2][sys_cat][sys_var].GetBinContent(ibinshape+1)-input_histograms_default[ibin2].GetBinContent(ibinshape+1)
-                        if error >= 0 :
+                        if error >= 0:
                             if self.systematic_rms[sys_cat] is True:
                                 error_var_up+=error*error
                                 count_sys_up=count_sys_up+1
                             else:
-                                if error > error_var_up :
+                                if error > error_var_up:
                                     error_var_up=error
                         else:
                             if self.systematic_rms[sys_cat] is True:
-                                if self.systematic_rms_both_sides[sys_cat] is True :
+                                if self.systematic_rms_both_sides[sys_cat] is True:
                                     error_var_up+=error*error
                                     count_sys_up=count_sys_up+1
                                 else:
                                     error_var_down+=error*error
                                     count_sys_down=count_sys_down+1
                             else:
-                                if abs(error) > error_var_down :
+                                if abs(error) > error_var_down:
                                     error_var_down = abs(error)
                     if self.systematic_rms[sys_cat]  is True:
-                        if count_sys_up is not 0:
+                        if count_sys_up != 0:
```

Comment:
Just a hint: Here you can just do `if count_sys_up:`

Everything evaluates to `True` except for `False`, `None`, `0`, or empty lists, tuples, dictionaries (hope I haven't forgotten something ;) )